services/mbin/mbin.service.js

```diff
@@ -0,0 +1,70 @@
+import Joi from 'joi'
+import { metric } from '../text-formatters.js'
+import { BaseJsonService, InvalidParameter, pathParams } from '../index.js'
+
+const schema = Joi.object({
+  subscriptionsCount: Joi.number().required(),
+}).required()
+
+export default class Mbin extends BaseJsonService {
+  static category = 'social'
+
+  static route = {
+    base: 'mbin',
+    pattern: ':magazine',
+  }
+
+  static openApi = {
+    '/mbin/{magazine}': {
+      get: {
+        summary: 'Mbin',
+        description:
+          'Mbin is a fork of Kbin, a content aggregator for the Fediverse.',
+        parameters: pathParams({
+          name: 'magazine',
+          description: 'The magazine to query.  This is CASE SENSITIVE.',
+          example: 'teletext@fedia.io',
+        }),
+      },
+    },
+  }
+
+  static defaultBadgeData = { label: 'magazine', namedLogo: 'activitypub' }
```

Comment:
A bit of a shame that we can't use a better logo that that of the underlying protocol. But to be able to add it to simple-icons, we'd need to meet the following requirement:

> If the brand is a Fediverse project, it needs to have a minimum of 10,000 Monthly Active Users as listed on [FediDB](https://fedidb.org/software).

Mbin [is over an order of magnitude under](https://fedidb.org/software/mbin) at this point.